integrations-and-supported-tools/sacred/notebooks/Neptune_Sacred.ipynb

```diff
@@ -247,145 +304,140 @@
     "        optimizer.step()\n",
     "\n",
     "    return {\"final_loss\": loss.item(), \"final_acc\": acc.cpu().item()}"
-   ],
-   "outputs": [],
-   "metadata": {
-    "id": "FOJvPNcp8cpO"
-   }
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "### Run your experiment and explore metadata in Neptune UI\n",
+    "## Run your experiment and explore metadata in Neptune UI\n",
     "\n",
     "All metadata is logged automatically to Neptune. \n",
     "\n",
     "After running your script or notebook cell you will get a link similar to:\n",
     "https://app.neptune.ai/o/common/org/sacred-integration/e/SAC-1\n",
     "with common/sacred-integration replaced by your project, and SAC-1 replaced by your run.\n",
     "\n",
-    "Click on the link to open the Run in Neptune to watch your model training live."
-   ],
-   "metadata": {
-    "id": "TH0t2ry48kXN"
-   }
+    "Click on the link to open the run in Neptune to watch your model training live."
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
    "source": [
     "ex.run()"
-   ],
-   "outputs": [],
-   "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/",
-     "height": 194,
-     "referenced_widgets": [
-      "4ff2f58c889445a891e93fdbf4801832",
-      "1c4cbf83896a4aa884f72a151fefab7e",
-      "f6f628a577734f58afd93ba81c280376",
-      "b046bff22c3e4fb68cce386b977258b6",
-      "9a87108dd16f4b61b8fe34cbbf6a170a",
-      "df861e99463545d7ac84cebde39368a6",
-      "009779801ecd44afa2f524ae371c6b2f",
-      "1cfea757feb546e39fc69a80f53f86cb"
-     ]
-    },
-    "id": "_0CWFytQwapG",
-    "outputId": "9b4d8474-e341-479d-f10a-fccc123fba64"
-   }
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
-    "## More Options"
+    "# More Options"
```

Comment:
is this intentional?

Reply:
yup.. separating log artifacts from the basic example

Reply:
What I mean is using a level 1 Markdown heading in the middle of a file. This change caught my attention, but I can see now that H1s are used throughout the file. If I'm not mistaken, usually H2 is used for section headings in our `.ipynb` files. Could you verify and standardize?

Reply:
Updated. The notebook title is supposed to be the only H1, that was missing here